listenbrainz/listenstore/tests/util.py

```diff
@@ -19,3 +24,16 @@ def generate_data(test_user_id, from_ts, num_records):
 
 def to_epoch(date):
     return int((date - datetime.utcfromtimestamp(0)).total_seconds())
+
+
+def create_test_data_for_influxlistenstore(user_name):
```

Comment:
A docstring for this function would be great. :)

Reply:
We try to use something similar to  [Google style docstrings](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments), [example](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/db/user.py#L93)